src/components/CreatePostForm.js

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from "react";
+import { v4 as uuid } from "uuid";
+import { useNavigate, useParams } from "react-router-dom";
+import {
+  getPostsFromLocalStorage,
+  savePostsToLocalStorage,
+} from "../utils/localStorage";
+import "../styles/createPost.css";
+import { blogThemesAvailable, readerGroups } from "../constants";
+
+const CreatePostForm = () => {
+  const navigate = useNavigate();
+  const { postId } = useParams();
+
+  const getInitialFormData = () => ({
+    id: uuid(),
+    title: "",
+    author: "",
+    themes: [],
+    readerGroup: "",
+    content: "",
+    image: "",
+    likes: 0,
+  });
+
+  const getPostData = (postId) => {
+    const posts = getPostsFromLocalStorage();
```

Comment:
store the posts in redux store no need of getting it from local storage everytime

Reply:
done